### PR TITLE
Rct poster session news

### PR DESCRIPTION
**Pull request by rct225**
First draft of news item for IRIS-HEP Poster session at Princeton

### CODE REVIEW

File: _posts/2020-02-27-iris-hep-posters-2020.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+{% include figure.html
+    file="/assets/images/posts/20200227-iris-hep-posters-2020-Schreiner-boost-histogram-extend-lhcb.jpg"
+    alt="Extending the Physics Reach of LHCb in Run 3 and Boost-Histogram for Analysis System"
+    caption="Henry Schreiner with posters for Extending the Physics Reach of LHCb in Run 3 Using Machine Learning in the
+Real-Time Data Ingestion and Reduction System and Boost-Histogram for Analysis System Photo Credit: Henry Schreiner."
+%}
+
+On Thursday, February 27, 2020 members of IRIS-HEP got together at Princeton University to take stock of our progress in the first 18 months and plan for the future. A poster session was held in conjunction with this review session to highlight the work of members from the first 18 months of IRIS-HEP.
+
+In all, twenty seven posters were presented by PIs, Post Docs, Students, Fellows and Staff encompassing topics from all areas of IRIS-HEP.  The topics included general information about IRIS-HEP, operational and policy information,  the development and deployment of services and tools, and the application of novel approaches to computing challenges.
```
Comment (henryiii) [inline _posts/2020-02-27-iris-hep-posters-2020.md]:
```suggestion
In all, twenty-seven posters were presented by PIs, Postdocs, Students, Fellows, and Staff encompassing topics from all areas of IRIS-HEP.  The topics included general information about IRIS-HEP, operational and policy information,  the development and deployment of services and tools, and the application of novel approaches to computing challenges.
```

File: _posts/2020-02-27-iris-hep-posters-2020.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+%}
+
+{% include figure.html
+    file="/assets/images/posts/20200227-iris-hep-posters-2020-Schreiner-boost-histogram-extend-lhcb.jpg"
+    alt="Extending the Physics Reach of LHCb in Run 3 and Boost-Histogram for Analysis System"
+    caption="Henry Schreiner with posters for Extending the Physics Reach of LHCb in Run 3 Using Machine Learning in the
+Real-Time Data Ingestion and Reduction System and Boost-Histogram for Analysis System Photo Credit: Henry Schreiner."
+%}
+
+On Thursday, February 27, 2020 members of IRIS-HEP got together at Princeton University to take stock of our progress in the first 18 months and plan for the future. A poster session was held in conjunction with this review session to highlight the work of members from the first 18 months of IRIS-HEP.
```
Comment (henryiii) [inline _posts/2020-02-27-iris-hep-posters-2020.md]:
```suggestion
On Thursday, February 27, 2020, members of IRIS-HEP got together at Princeton University to take stock of our progress in the first 18 months and plan for the future. A poster session was held in conjunction with this review session to highlight the work of members from the first 18 months of IRIS-HEP.
```

File: _posts/2020-02-27-iris-hep-posters-2020.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    file="/assets/images/posts/20200227-iris-hep-posters-2020-Gray-ACTS.jpg"
+    alt="Tracking with A Common Tracking Software (ACTS)"
+    caption="Heather Gray with the poster for Tracking with A Common Tracking Software (ACTS) Photo Credit: Heather Gray."
+%}
+
+{% include figure.html
+    file="/assets/images/posts/20200227-iris-hep-posters-2020-Schreiner-boost-histogram-extend-lhcb.jpg"
+    alt="Extending the Physics Reach of LHCb in Run 3 and Boost-Histogram for Analysis System"
+    caption="Henry Schreiner with posters for Extending the Physics Reach of LHCb in Run 3 Using Machine Learning in the
+Real-Time Data Ingestion and Reduction System and Boost-Histogram for Analysis System Photo Credit: Henry Schreiner."
```
Comment (henryiii) [inline _posts/2020-02-27-iris-hep-posters-2020.md]:
```suggestion
Real-Time Data Ingestion and Reduction System and Boost-Histogram for Analysis Systems. Photo Credit: Henry Schreiner."
```

**Comment by henryiii** [comment]
If you don't mind, I'll probably merge my CI change before this and then use this to verify the new CI builds the site.

**Comment by rct225** [comment]
I don’t mind.  Go ahead.

Rob

**Pull request merged**